CSS_Tasks/CSS_Advanced/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <title>WebGL</title>
+    <link href="stylesheet.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <button type="submit" class="button buttonGreenStyle"> Вход </button>
+        <button type="submit" class="button buttonGreenStyle"> Регистрация </button>
+    </header>
+
+    <main class="pageContainer">
+        <div class="navigationBar">
+
+            <nav>
+                <ul>
+                    <li>Компютърна графика с WebGL </li>
+                    <hr>
+                    <li>Програмиране с Gо</li>
+                    <hr>
+                    <li>Програмиране с Ruby</li>
+                    <hr>
+                    <li>Agile</li>
+                    <hr>
+                </ul>
+            </nav>
+
+        </div>
+        <div class="content">
+
+            <h1> Компютърна графика с WebGL</h1>
+            <hr>
+
+            <h2>доц. П. Бойчев</h2>
+            <p>След завършване на курса студентите ще могат да създават тримерни статични изображения и тримерни анимации</p>
+
+            <p><i>Тип: </i> задължителна избираема</p>
+            <p><i>Курс: </i> 1 <sup>ви</sup></p>
+            <p><i>Програма: </i> бакалавър</p>
+
+            <textarea placeholder="Добавяне на коментар"></textarea>
+            <div class="mainButtons">
+                <button type="submit" class="button buttonWhiteStyle"> Отказ </button>
+                <button type="submit" class="button buttonGreenStyle"> Публикуване </button>
+            </div>
+
+            <h2>Коментари</h2>
+
+            <div>
+
+                <article>
+
+                    <div class="commentInfo">
+                        <p class="commentAuthor">Вили Велева</p>
+                        <p>07.11.2015</p>
+                    </div>
+                    <p>"Препоръчвам записването на курса. Интересни неща се учат, има и освобождаване с реферати."</p>
+                    <hr>
+                </article>
+                <article>
+
+                    <div class="commentInfo">
+                        <p class="commentAuthor">Вили Велева</p>
```

Comment:
`p` is for paragraphs. This means most of the time the reader / developer would expect that each of these items is on a new line on the screen. You may want to switch to `div`s or `span`s here.